### PR TITLE
Move identical VisualOdometry update method into that class from Rover/Copter

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: libraries/AP_VisualOdom/AP_VisualOdom.h [hunk 1]
```diff
@@ -49,29 +49,28 @@ class AP_VisualOdom
         Vector3f position_delta;    // position delta (in meters) of most recent update
         uint64_t time_delta_usec;   // time delta (in usec) between previous and most recent update
         float confidence;           // confidence expressed as a value from 0 (no confidence) to 100 (very confident)
-        uint32_t last_update_ms;    // system time (in milliseconds) of last update from sensor
+        uint32_t last_sensor_update_ms;    // system time (in milliseconds) of last update from sensor
+        uint32_t last_sensor_update_ms_i_processed; // timestamp of last sensor update that was processed
```
Comment (rmackay9) [inline libraries/AP_VisualOdom/AP_VisualOdom.h]: sorry to be picking but can we rename this variable from "last_sensor_update_ms_i_processed" to "last_processed_sensor_update_ms" or similar?

**Comment by rmackay9** [comment]
looking good, it's been tested in SITL at least?  It's rarely enough used that if we broke it it would take a while to realise.

File: libraries/AP_VisualOdom/AP_VisualOdom.h (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_VisualOdom/AP_VisualOdom.h]: Fixed and fixed.

**Comment by peterbarker** [comment]
@rmackay9 Copter's autotest relies on this functionality for it's VisionPosition test.  There is currently no equivalent for Rover, however.

**Comment by rmackay9** [comment]
Great thanks.  I'll merge once it passes travis.

**Pull request merged**